PluginsAndFeatures/azure-toolkit-for-intellij/rider/src/com/microsoft/intellij/runner/appbase/config/runstate/AppDeployStateUtil.kt

```diff
@@ -141,14 +142,16 @@ object AppDeployStateUtil {
                 event.setResult(it)
             }
 
-            if (publishableProject.isDotNetCore) {
-                publishService.invokeMsBuild(publishableProject, listOf(targetProperties), false, true, true, onFinish)
-            } else {
-                publishService.webPublishToFileSystem(publishableProject.projectFilePath, outPath, false, true, onFinish)
+            Lifetime.Eternal.launchIOBackground {
```

Comment:
It does not looks like we need to run this on AWS. We could move the whole thread to background I guess (no need in application.invokeLater { ... } here).